wodles/aws/aws_s3.py

```diff
@@ -3883,6 +3938,14 @@ def get_script_arguments():
     if parsed_args.iam_role_duration is not None and parsed_args.iam_role_arn is None:
         raise argparse.ArgumentTypeError('Used --iam_role_duration argument but no --iam_role_arn provided.')
 
+    if parsed_args.subscriber is not None:
```

Comment:
The parameters required by Security Lake should be validated inside the `AWSSQSQueue` class since another type of `subscribers` could require another set of mandatory parameters.

Reply:
Moved validation inside `AWSSQSQueue`.

https://github.com/wazuh/wazuh/blob/3c4c0d60b9a5a22fc6a40e7d137a6f4e948a2078/wodles/aws/aws_s3.py#L3225-L3247